Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
+
+    bin.install "_bin/bitrise"
+  end
+
+  test do
+    output = shell_output(bin/"bitrise --version")
```

Comment:
No the code is not available anymore. I’m not familiar with `bitrise`, but could we write a dummy `bitrise.yml` file then run `bitrise` and check it did execute the correct commands? We don’t need something complicated; we just want to check `bitrise` was correctly installed and works as expected.